README.md

```diff
@@ -227,21 +227,24 @@ For example: `export MINIO_SERVER_URL="https://minio.example.net"`
 `mc` provides a modern alternative to UNIX commands like ls, cat, cp, mirror, diff etc. It supports filesystems and Amazon S3 compatible cloud storage services. Follow the MinIO Client [Quickstart Guide](https://docs.min.io/docs/minio-client-quickstart-guide) for further instructions.
 
 # Upgrading MinIO
-MinIO server supports rolling upgrades, i.e. you can update one MinIO instance at a time in a distributed cluster. This allows upgrades with no downtime. Upgrades can be done manually by replacing the binary with the latest release and restarting all servers in a rolling fashion. However, we recommend all our users to use [`mc admin update`](https://docs.min.io/docs/minio-admin-complete-guide.html#update) from the client. This will update all the nodes in the cluster simultaneously and restart them, as shown in the following command from the MinIO client (mc):
+Upgrades require zero downtime in MinIO, all upgrades are non-disruptive, all transactions on MinIO are atomic. So upgrading all the servers simultaneously is the recommended way to upgrade MinIO.
 
+> NOTE: requires internet access to update directly from https://dl.min.io, optionally you can host any mirrors at https://my-artifactory.example.com/minio/
+
+- `mc admin update` updates and restarts all servers simultaneously, applications would retry and continue their respective operations upon upgrade. [`mc admin update`](https://docs.min.io/docs/minio-admin-complete-guide.html#update) 
```

Comment:
```suggestion

For deployments which installed the MinIO server binary by hand, use [`mc admin update`](https://docs.min.io/minio/baremetal/reference/minio-mc-admin/mc-admin-update.html)

For DEB/RPM installations, download the DEB/RPM release and install it as recommended by your operating system (e.g. `dnf` or `rpm`). `mc admin update` cannot update DEB/RPM releases.

```

Reply:
Noticed that legacy docs page for mc admin update conflicts with what is written above - I'll push something to clean that up, but after my next pass on the admin docs we should consider pulling down the "complete" guides and pushing users to the next gen docs.